libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalAnalysisView.tsx

```diff
@@ -116,10 +118,16 @@ export class CausalAnalysisView extends React.PureComponent<
         compositeFiltersRelabeled,
         new AbortController().signal
       );
-      this.setState({ currentGlobalCausalEffects: result.global_effects });
+      this.setState({
+        currentGlobalCausalEffects: result.global_effects.sort(
+          (d1, d2) => d2.point - d1.point
+        )
+      });
     } else {
       this.setState({
-        currentGlobalCausalEffects: this.props.data.global_effects
+        currentGlobalCausalEffects: this.props.data.global_effects.sort(
```

Comment:
can we have sorting in one function and send the global_effects as arg' while using it as we have sort in multiple places?

Reply:
Done